vtex/toolbelt
# Use new VTEXID login flow

#### How should this be manually tested?
```
yarn global add vtex@2.106.3-beta.2
```
Make sure you're able to login on Chrome, Firefox and Safari. We have to test it on and MacOS as well, I tested on Linux (Firefox and Chrome) and Windows (Chrome and Edge).

#### Types of changes

#### Chores checklist
